src/pivot-table/components/cells/styles.tsx

```diff
@@ -1,21 +1,30 @@
 import Box from "@mui/material/Box";
 import { styled } from "@mui/material/styles";
 import type { stardust } from "@nebula.js/stardust";
+import { HEADER_ICON_SIZE } from "../../constants";
+import { CELL_PADDING, textStyle } from "../shared-styles";
 
-interface Props {
+const baseFlex: React.CSSProperties = {
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+};
+
+interface StyledHeaderCellWrapperProps {
   interactions: stardust.Interactions;
   background: string;
   hoverBackground: string;
+  shouldShowMenuIcon: boolean;
 }
 
 export const StyledHeaderCellWrapper = styled(Box, {
```

Comment:
No really sure why get any value of out using `Box` component here. As it could be just a `div` instead.